packages/loopring_v3/contracts/amm/LoopringAmmPool.sol

```diff
@@ -98,9 +98,17 @@ contract LoopringAmmPool is
         external
         nonReentrant
     {
+        require(state.accountID == 0 || msg.sender == address(controller), "UNAUTHORIZED");
         state.setupPool(config);
     }
 
+    function enableExitMode()
+        external
+        onlyFromController
```

Comment:
now with  exit mode, maybe we should check `controller` is not 0x0 in the constructor.

Reply:
should we do enterExitMode(bool enter) so we can also exit from Exit mode?